style.css

```
@@ -6,6 +6,37 @@ body {
     left: 0;
 }
 
+#textMenu {
+  max-height: 4%;
+  max-width: 50%;
+  min-height: 4px;
+  min-width: 900px;
+  background-color: white;
+  box-shadow: 0 0 5px rgba(0,0,0,0.5);
+  border-radius: 20px;
+  padding: 10px 10px;
+  text-align:center;
+  position: absolute;
+  top: 95%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  overflow: auto;
+}
+
+#textIndicator {
+  background-color: white;
+  padding: 10px 10px;
```

```suggestion
  padding: 10px;
```
Это будет равносильно, если имелось ввиду другое, см. http://htmlbook.ru/css/padding